File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// JellyBellyWiki SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class JellyBellyWikiMakeContext
{
    public static function call(array $ctxmap, ?JellyBellyWikiContext $basectx): JellyBellyWikiContext
    {
        return new JellyBellyWikiContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# JellyBellyWiki SDK utility: make_context
require_relative '../core/context'
module JellyBellyWikiUtilities
  MakeContext = ->(ctxmap, basectx) {
    JellyBellyWikiContext.new(ctxmap, basectx)
  }
end
